### PR TITLE
Message handler

### PR DESCRIPTION
**Pull request by bzapa**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -79,23 +81,88 @@ def on_click(self, select=True):
...
         return future
 
 
+class GetFirmwareResponder(protocol.OnGetFirmware):
+
+    def __init__(self, firmware_service: FirmwareService):
+        self.firmware_service = firmware_service
+
+    def on_request(self, request) -> Future[protocol.FirmwareData]:
+        print("getFirmwareResponder")
```
Comment (bgrzesik) [inline main.py]:
```suggestion
        logging.debug("getFirmwareResponder")
```

File: server/target/const.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+BOARDS_PATH = "/home/pi/openocd/tcl/target"
```
Comment (bgrzesik) [inline server/target/const.py]: I'd rather prefer this file to be called 'constants.py'

File: server/target/files_repository.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# import glob
```
Comment (bgrzesik) [inline server/target/files_repository.py]: Is this file intentional?

Comment (bzapa) [reply server/target/files_repository.py]: deleted file

File: server/target/const.py (hunk 2 above)
Comment (bzapa) [reply server/target/const.py]: renamed

File: main.py (hunk 1 above)
Comment (bzapa) [reply main.py]: Removed. It was temporary for debug.

File: server/comm/transport/frame_coding.py [hunk 4]
```diff
@@ -13,7 +13,7 @@ def write_byte(self, byte: int):
             self._write_slice(False)
 
     def write_slice_size(self, size: int):
-        assert size <= 0x3FFF
+        assert size <= 0x3FFFF
```
Comment (bgrzesik) [inline server/comm/transport/frame_coding.py]: Oh there is mistake here, it's suppose to be
```suggestion
        assert size <= 0x7FFF
```
16 383 -> 32 767 bytes

**Review by bgrzesik — APPROVED** [review]

File: server/comm/transport/frame_coding.py (hunk 4 above)
Comment (bzapa) [reply server/comm/transport/frame_coding.py]: changed

Comment (bgrzesik) [reply server/comm/transport/frame_coding.py]: Did you tested this though?

Comment (bzapa) [reply server/comm/transport/frame_coding.py]: Yes, no issues

**Comment by bgrzesik** [comment]
Let's ignore CI for now :)
